spec/interactors/update_map_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+describe UpdateMap do
+  it "organizes flow of interactors for editing a map" do
+    expect(described_class.organized).to eq(
+      [
+        SaveMap,
+        BroadcastCurrentMap,
+        BroadcastMapSelector
+      ]
+    )
```

Comment:
Does this feel a little excessive to you? Do we think the outcomes of the organizers are covered by the system specs instead?

Reply:
I found it useful when test driving this to be able to lay out how I expected it to work, to collect by thoughts and then see a failure that guided the next step of writing the code.